.github/workflows/build-pgaf-enterprise-nightlies.yml

```diff
@@ -0,0 +1,81 @@
+name: Build and publish pgaf enterprise nightly packages
+
```

Comment:
I guess we will get rid of travis job after merging this pr, right ?

Reply:
Yes I will delete the nightly job description. But it's just a description not file